src/cljc/ataru/tarjonta/haku.cljc

```diff
@@ -18,3 +22,53 @@
   [haku]
   (some-> (:hakutapa-uri haku)
           (string/starts-with? hakutapa-jatkuva-haku)))
+
+; In case of peruskoulun jälkeisen koulutuksen yhteishaku, it's specified
```

Comment:
Onko näitä siirrettyjä funkkareita tarkoitus valjastaa frontin käyttöön? Ehkä osa näistä voisi olla myös tyyliin filussa hakijan-lahtokoulu tai vastaavaa, kun eivät ehkä suoraan hakuun liity